lib/mayaUsd/ufe/UsdTransform3dFallbackMayaXformStack.cpp

```diff
@@ -80,7 +80,12 @@ bool MatchingSubstack(const std::vector<UsdGeomXformOp>& ops)
     // current op's index, since each index can appear only once.
     int orderedNdx = UsdTransform3dMayaXformStack::NdxTranslate;
     for (const auto& op : ops) {
-        auto opNdx = gOpNameToNdx.at(op.GetOpName());
+        auto found = gOpNameToNdx.find(op.GetOpName());
+        if (found == gOpNameToNdx.end()) {
+            // Found an op that doesn't match a Maya fallback stack op.
```

Comment:
The Maya fallback transform stack must be the last set of transform ops in a prim's transform stack.  If there is already a Maya fallback transform stack at the tail of a prim's transform stack, any op that does not match means the Maya fallback transform stack has been "corrupted" in some way by adding or inserting an op that shouldn't be there.

If that happens, we don't create a fallback to the fallback: we detect the erroneous transform op, and return a nullptr Transform3d interface.